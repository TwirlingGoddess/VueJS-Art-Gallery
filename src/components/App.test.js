import { shallow } from 'vue-test-utils'
import App from './App.vue'
import Gallery from './Gallery.vue'
const mock = 'view-resource-mock'

describe('App', () => {
  it('matches snapshot without images', () => {
    const wrapper = shallow(App) 
    expect(wrapper).toMatchSnapshot()
  })

  it('matches snapshot with images', (done) => {
    const wrapper = shallow(App)
    wrapper.setData({images: [{image:'https://nrs.harvard.edu/urn-3:HUAM:CARP09302_dynmc', caption: '', id: 7}] })
    wrapper.vm.$nextTick(() => {
      expect(wrapper).toMatchSnapshot()
      done()
    })
  })
})
