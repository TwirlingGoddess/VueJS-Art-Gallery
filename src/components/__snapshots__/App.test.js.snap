// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<div id="app">
  <!---->
</div>
`;

exports[`App matches snapshot with images 1`] = `
<div id="app">
  <!---->
</div>
`;

exports[`App matches snapshot without images 1`] = `
<div id="app">
  <!---->
</div>
`;
